.github/workflows/build.yml

```diff
@@ -83,6 +83,14 @@ jobs:
         run: python .github/scripts/checkstyle.py
         env:
           GITHUB_WORKSPACE: ${{ github.workspace }}
+      - name: Update Floating 'cached' Tag on Successful Trunk Build
+        if: github.ref == 'refs/heads/trunk' && success()
+        run: |
+          git fetch --tags
+          git tag -f cached $GITHUB_SHA
```

Comment:
When we create the new tag, can we add a message like "Automatically generated after running workflow 1234. https://github.com/fonsdant/github-actions/actions/runs/10902593757" or similar? This will help us if we need to debug caching issues

Reply:
Updated!